src/internal/utils/tests/sort.spec.ts

```diff
@@ -345,6 +345,14 @@ describe("createSorts", () => {
                 ).toEqual([]);
             });
         });
+
+        describe("heatmap", () => {
+            it("should return empty array", () => {
```

Comment:
better describe this case. Sounds like it should return empty array every time

Reply:
I glanced over other tests in this commit and it seem those descriptions need bit of a polish as well. Try to describe them like if you are explaining behavior. `it should return empty array if there are no custom sorts and ...`